deployments/rainbow-brass/src/main/java/org/sa/rainbow/brass/model/map/MapTranslator.java

```diff
@@ -186,7 +188,7 @@ public static String generateLocationConstants(){
         buf+="const "+INITIAL_ROBOT_LOCATION_CONST+";\n";
         buf+="const "+TARGET_ROBOT_LOCATION_CONST+";\n\n";
         buf+="formula "+GOAL_PRED_DEF+"\n\n";
-        buf+="formula "+STOP_PRED_DEF+"\n\n";
+        buf+="formula "+STOP_PRED + " = "+GOAL_PRED+(considerBattery?" | "+ROBOT_BATTERY_VAR+"<"+ROBOT_BATTERY_DELTA:"")+";"+"\n\n";
```

Comment:
Why is "<" here? Intuitively I expected ">=", but I'm having a hard time compiling these lines into a prism code.

Reply:
@javier-camara Do you know?